### PR TITLE
chore(): Update app and tabs

### PR DESCRIPTION
**Pull request by mhartington**
- Updates app to beta 18
- Updates to new tabs setup
- Adds firebase hosting
- Adds h2-push generator script

### CODE REVIEW

File: src/app/pages/speaker-detail/speaker-detail.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <ion-header>
   <ion-toolbar>
     <ion-buttons slot="start">
-      <ion-back-button text="Speakers"></ion-back-button>
+      <ion-back-button defaultHref="/app/tabs/(speakers:speakers)"></ion-back-button>
```
Comment (manucorporat) [inline src/app/pages/speaker-detail/speaker-detail.html]: needs update

File: .firebase/hosting.d3d3.cache [hunk 2]
```diff
@@ -0,0 +1,908 @@
+100.2993ce39965c8abca50e.js,1544629057856,d2c717604d231e64fbc69b24b109991146a9e4d3b88fdd5c75250d2d6f3e7463
```
Comment (manucorporat) [inline .firebase/hosting.d3d3.cache]: Probably this file changes a lot between builds, better to .gitignore

**Comment by mhartington** [comment]
@manucorporat updated based on review

**Comment by brandyscarney** [comment]
Can this be updated to beta.19 and merged?

**Comment by mhartington** [comment]
done!

**Pull request merged**